hub/api/tests/test_api.py

```diff
@@ -0,0 +1,75 @@
+import os, shutil
+import numpy as np
+from hub.api.dataset import Dataset
+from hub.core.chunk_engine.read import read_dataset_meta, read_tensor_meta
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def clear_test_files():
+    shutil.rmtree("/tmp/hub-test/", ignore_errors=True)
```

Comment:
since the pytest-options PR was merged, you can get rid of this function `clear_test_files`, it should automatically clear storage provider fixtures